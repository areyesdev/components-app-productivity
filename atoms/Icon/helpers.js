const ICONS_ASSETS_PATH = '/icons',
  iconMap = {
    arrowRight: 'arrow-right',
    angleDown: 'angle-down',
    iconPlus: 'icon-plus',
  },
  iconSize = {
    sm: 15,
    md: 26,
    lg: 40,
    xl: 40,
  }

export const mapType = (type) => `${ICONS_ASSETS_PATH}/${iconMap[type]}.svg`
export const mapSize = (size) => iconSize[size]
