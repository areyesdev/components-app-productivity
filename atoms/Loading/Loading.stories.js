import { Loading, styles } from '.'

import { getTemplate } from '../../helpers/storybook'

const Template = getTemplate(Loading, styles)

export default {
  title: 'Atoms/Loading',
  component: Loading,
  args: {
    isPlayground: true,
    children:
      "Adventures washed awfully web poured pirate IÃ¡ve extend shared? Glows visit bitter fire-breather 400 triumphant splash responsible Oakenshield's prong invitations mortal? It must be taken deep into Mordor and cast back into the fiery chasm from whence it came.",
  },
}

export const Default = Template.bind({})
