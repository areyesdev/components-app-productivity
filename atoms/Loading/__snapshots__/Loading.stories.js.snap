// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Loading Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="loading"
    />
  </div>
</div>
`;
